pkg/scheduler/framework/plugins/nodeaffinity/node_affinity.go

```diff
@@ -145,11 +145,6 @@ func (pl *NodeAffinity) Score(ctx context.Context, state *framework.CycleState,
 		return 0, framework.AsStatus(fmt.Errorf("getting node %q from Snapshot: %w", nodeName, err))
 	}
 
-	node := nodeInfo.Node()
```

Comment:
This line can't be removed, it is used below.

Reply:
Thanks, revised.